index.js

```diff
@@ -1 +1,29 @@
-console.log('hello world');
+import express from 'express';
+import { read } from './jsonFileStorage.js';
+
+const PORT = 3004;
+
+const app = express();
+
+const whenIncomingRequestIndex = (req, res) => {
```

Comment:
consider naming your function with what it does rather instead

```suggestion
const handleRecipeRequest = (req, res) => {
```